.idea/zulip-flutter.iml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/.pub" />
+      <excludeFolder url="file://$MODULE_DIR$/build" />
+      <excludeFolder url="file://$MODULE_DIR$/.dart_tool" />
+      <excludeFolder url="file://$MODULE_DIR$/packages/zulip_plugin/.pub" />
+      <excludeFolder url="file://$MODULE_DIR$/packages/zulip_plugin/build" />
+      <excludeFolder url="file://$MODULE_DIR$/packages/zulip_plugin/.dart_tool" />
+      <excludeFolder url="file://$MODULE_DIR$/macos/Flutter/ephemeral" />
+      <excludeFolder url="file://$MODULE_DIR$/ios/.symlinks" />
+      <excludeFolder url="file://$MODULE_DIR$/linux/flutter/ephemeral" />
+      <excludeFolder url="file://$MODULE_DIR$/windows/flutter/ephemeral" />
+    </content>
+    <orderEntry type="inheritedJdk" />
```

Comment:
This element is absent in the file on my home desktop. I'm not sure what it means.

Reply:
Yeah, I'm not sure either. Maybe leave it out, and if we need it we'll eventually find out?